utils.go

```diff
@@ -44,49 +61,22 @@ func SelectHopHints(ctx context.Context, lnd *lndclient.LndServices,
 	// through GetChanInfo
 	chanInfoCache := make(map[uint64]*lndclient.ChannelEdge)
 
-	// skipCache is a simple cache which holds the indice of any
-	// channel we've added to final hopHints
+	// skipCache is a simple cache which holds the indices of any
+	// channel we've added to final hopHints.
```

Comment:
Does this comment still hold? Seems that we also add private nodes to `skipCache`, and we're not using them for final hop hints?

Reply:
Yes the comment was a bit outdated, PTAL